### ISSUE TITLE
Printable CSS

### ISSUE THREAD

**Issue by albertobasso88** [troolee/gridstack.js#300]
Hello,
I want to print gridstack the way is it shown in the browser, but when I print it, all the divs are stacked together. Is there a way to solve this?

**Comment by jlowcs** [comment]
Not really. The reason is that the divs have absolute positionning.

Here's how I solved this problem:
I added a "print" button to my app. That button changes the layout a bit then lauches the print.
The layout is changed so that:
- in JS
 * the widgets are ordered in the DOM
 * each widget has a fixed width (I chose 8.5in)
- in CSS
 * each widget is set to:
```css
display: block;

position: static;

top: 0!important;
left: 0;
right: 0;

height: auto!important;
```

**Comment by albertobasso88** [comment]
Thanks!
